test/test_podman_baseline.sh

```diff
@@ -72,7 +72,7 @@ podman rmi --all
 ########
 # Create Fedora based image
 ########
-image=$(podman pull fedora)
+image=$(podman pull fedora:28)
```

Comment:
So F29/latest doesn't work on multi-arch?  Would you be OK with changing this to a variable that we could send the value to from the command line and if not given, default to 28?

Reply:
I have a couple of issues with this ... first, I think we should guarantee we are getting the real fedora image by using the fq name. I also think we should be testing the latest image,  by either providing the latest tag or not providing a tag.

Reply:
Federa image is used from `egistry.fedoraproject.org/fedora:29` and now all images names are replaces with their respective fq name.

Reply:
Now using `registry.fedoraproject.org/fedora:latest`